src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -753,7 +753,6 @@
           "description": "The set of results contained in an SARIF log. The results array can be omitted when a run is solely exporting rules metadata. It must be present (but may be empty) in the event that a log file represents an actual scan.",
           "type": "array",
           "minItems": 0,
-          "uniqueItems": true,
```

Comment:
Because the AndroidStudio converter generates non-unique items, and we're no longer using HashSets, we have two choices:

1. Remove `uniqueItems` from the `results` array.
2. Check each result for equality with all existing results before adding to the list (N^2 behavior).